src/TimePicker/ClockNumber.js

```diff
@@ -45,6 +45,8 @@ function getStyles(props, context) {
     [-54.5, 19.6],
   ];
 
+  const positions = muiTheme.isRtl ? originalPositions.map(([x, y]) => [-x, y]) : originalPositions;
```

Comment:
I'm gonna make the same comment as here https://github.com/callemall/material-ui/pull/5001#discussion_r75089972?